Fix cplen calculation error in ff_hook_getsockname

zhaozihanzzh opened the pull request:
Dear authors,
Thank you for developing and maintaining f-stack. Currently I'm trying to use f-stack's LD_PRELOAD function in my research. However, I encounter several problems. While debugging, I found that the cplen seems to be wrong in ff_hook_getsockname.
By the way, now I'm still stuck at ff_hook_epoll_wait. It seems that ff_hook_epoll_wait can't get notified on the corresponding events when running a complicated application(possibly with several threads created with clone syscall). Any ideas?

zhaozihanzzh committed:
Fix cplen calculation

jfb8856606 commented on adapter/syscall/ff_hook_syscall.c:
```diff
@@ -419,7 +419,7 @@ ff_hook_getsockname(int fd, struct sockaddr *name,
 
     if (ret == 0) {
-        socklen_t cplen = *namelen ? *sh_namelen > *namelen
+        socklen_t cplen = *namelen < *sh_namelen ? *namelen
```
Can you also modify the code in ff_hook_getpeername?

zhaozihanzzh committed:
Fix cplen calculation errors in ff_hook_syscall.c
This commit also modifies the style of the previous fix to be in tune with the original ff_hook_accept.
